client/post-editor/media-modal/index.jsx

```diff
@@ -210,7 +210,7 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorDone( error, blob, imageEditorProps ) {
+	onImageEditorDone( error, blob, imageEditorProps, media ) {
```

Comment:
We could also clarify that media.id is an optional prop that will be parroted back on callbacks.